src/main/java/org/broadinstitute/dsm/route/ClinicalKitsRoute.java

```diff
@@ -0,0 +1,87 @@
+package org.broadinstitute.dsm.route;
+
+import lombok.NonNull;
+import org.apache.commons.lang3.StringUtils;
+import org.broadinstitute.ddp.db.TransactionWrapper;
+import org.broadinstitute.dsm.db.DDPInstance;
+import org.broadinstitute.dsm.db.dao.kit.BSPKitDao;
+import org.broadinstitute.dsm.db.dto.kit.ClinicalKitDto;
+import org.broadinstitute.dsm.model.ParticipantWrapper;
+import org.broadinstitute.dsm.model.bsp.BSPKitInfo;
+import org.broadinstitute.dsm.model.bsp.BSPKitQueryResult;
+import org.broadinstitute.dsm.statics.RequestParameter;
+import org.broadinstitute.dsm.util.ElasticSearchUtil;
+import org.broadinstitute.dsm.util.NotificationUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import spark.Request;
+import spark.Response;
+import spark.Route;
+
+import java.sql.Connection;
+import java.util.Map;
+import java.util.Optional;
+
+public class ClinicalKitsRoute implements Route {
+    private String FIRSTNAME = "firstName";
+    private String LASTNAME = "lastName";
+    private String DATE_OF_BIRtH = "dateOfBirth";
+
+    private static final Logger logger = LoggerFactory.getLogger(ClinicalKitsRoute.class);
+
+    private NotificationUtil notificationUtil;
+
+    public ClinicalKitsRoute(@NonNull NotificationUtil notificationUtil) {
+        this.notificationUtil = notificationUtil;
+    }
+
+    @Override
+    public Object handle(Request request, Response response) {
+        String kitLabel = request.params(RequestParameter.LABEL);
+        if (StringUtils.isBlank(kitLabel)) {
+            throw new RuntimeException("Please include a kit label as a path parameter");
+        }
+
+        return TransactionWrapper.inTransaction(conn -> {
+            return getClinicalKit(conn, kitLabel, response);
+        });
+    }
+
+    private ClinicalKitDto getClinicalKit(Connection conn, String kitLabel, Response response) {
```

Comment:
it is not necessary to pass response here, you could manipulate response in upper level, in `handle` method according to the returned result of this method.